WcaOnRails/app/models/poll_option.rb

```diff
@@ -0,0 +1,9 @@
+class PollOption < ActiveRecord::Base
+
+  belongs_to :poll
+
+  has_many :votes, dependent: :destroy
+  has_many :users, through: :votes
```

Comment:
Do you mean a poll_option?
We don't now, but we may in the future :) (like knowing who voted for which option)